docs/kms/README.md

```diff
@@ -474,14 +474,19 @@ head -c 32 /dev/urandom | xxd -c 32 -ps
 Alternatively, you may pass a master key as a [Docker secret](https://docs.docker.com/engine/swarm/secrets/).
 
 ```bash
-echo "my-minio-key:6368616e676520746869732070617373776f726420746f206120736563726574" | docker secret create kms_master_key
+echo "my-minio-key:6368616e676520746869732070617373776f726420746f206120736563726574" | docker secret create sse_master_key
 ```
 
 To use another secret name, follow the instructions above and replace `kms_master_key` with your custom names (e.g. `my_kms_master_key`).
-Then, set the `MINIO_KMS_MASTER_KEY_FILE` environment variable to your secret name:
+Then, set the `MINIO_SSE_MASTER_KEY_FILE` environment variable to your secret name:
 
 ```bash
-export MINIO_KMS_MASTER_KEY_FILE=my_kms_master_key
+export MINIO_SSE_MASTER_KEY_FILE=my_kms_master_key
+```
+
+To enable auto-encryption set the environment variable to `on`:
```

Comment:
Auto-Encryption is documented in Appendix A. See: https://github.com/minio/minio/pull/8975/files#diff-e470785eff957ab411272c52f473356cL425

Reply:
however the auto encryption using `MINIO_KMS_AUTO_ENCRYPTION` doesn't work with the master key if it's set via `MINIO_SSE_*` prefix, I had to switch to `MINIO_SSE_AUTO_ENCRYPTION ` for it to work